scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -185,3 +228,131 @@ task CreateImportTsvs {
   }
 }
 
+# Creates all the tables necessary for the LoadData operation
+# As an optimization, I also generate a (table, dir, files) csv file which contains
+# most of inputs necessary for the following LoadTable task.
+task CreateTables {
+	meta {
+    	volatile: true
+  	}
+
+	input {
+      String project_id
+      String dataset_name
+      String storage_location
+      String datatype
+      Int max_table_id
+      File schema
+      String numbered
+      String partitioned
+      String uuid
+      Array[String] tsv_creation_done
```

Comment:
what does this do?

Reply:
by requiring this as an input, this ensures that CreateImportTSVs runs before CreateTables can start. (removed here but added in to LoadTables)